Makefile

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html ;)

Reply:
Added help to Makefile in next commit